scripts/qa_crowbarsetup.sh

```diff
@@ -985,6 +985,17 @@ EOF
         rm -f $pfile
         crowbar ntp proposal commit default
     fi
+    if [[ $keep_existing_hostname = 1 ]] ; then
```

Comment:
wouldn't it make more sense to add a `provisioner` case to `custom_configuration()`? I think there is no need to run this much code here. this should be done by `proposal_set_value` etc